bundle/store.go

```diff
@@ -234,6 +234,7 @@ type ActivateOpts struct {
 	ExtraModules map[string]*ast.Module // Optional
 
 	legacy bool
+	Schema interface{}
```

Comment:
Since the compiler is provided as a parameter to the activate call, I think we may as well configure the schemas on it before calling activate. This file/package are complicated due to legacy reasons--I'd like to avoid changing it if possible.

Reply:
Done